vm-virtio/src/iommu.rs

```diff
@@ -330,6 +336,8 @@ impl Request {
         avail_desc: &DescriptorChain,
         mem: &GuestMemoryMmap,
         mapping: &Arc<IommuMapping>,
+        ext_mapping: &BTreeMap<u32, Arc<dyn ExternalDmaMapping>>,
+        ext_domain_mapping: &mut BTreeMap<u32, Arc<dyn ExternalDmaMapping>>,
```

Comment:
Those 2 have exactly the same prototype, it'd be worth adding some documentation to explain what's different between them.
And maybe call the first one `ext_endpoint_mapping`?

Reply:
Yes you're right.

Reply:
Done